app/src/main/java/org/simple/clinic/monthlyReports/questionnaire/entry/epoxy/model/SeparatorEpoxyModel.kt

```diff
@@ -0,0 +1,19 @@
+package org.simple.clinic.monthlyReports.questionnaire.entry.epoxy.model
+
+import android.annotation.SuppressLint
+import android.view.View
+import com.airbnb.epoxy.EpoxyHolder
+import com.airbnb.epoxy.EpoxyModelClass
+import com.airbnb.epoxy.EpoxyModelWithHolder
+import org.simple.clinic.R
+
+@SuppressLint("NonConstantResourceId")
+@EpoxyModelClass(layout = R.layout.view_questionnaire_section_separator)
```

Comment:
Maybe call this `view_questionnaire_separator`

Reply:
@olttwa Renamed the file to `view_questionnaire_vertical_separator`